test/lint/test_runner/src/main.rs

```diff
@@ -650,6 +724,10 @@ fn main() -> ExitCode {
     };
 
     let git_root = get_git_root();
+    let commit_range = commit_range();
+    let commit_log = check_output(git().args(["log", "--no-merges", "--oneline", &commit_range]))
+        .expect("git error");
```

Comment:
nit: 
In `commit_range` there is also a `.expect` which has the text "git command failed"
it would make sense to have the same text for consistency, but feel free to ignore

Reply:
yea should be fine just thought I'd point it out, I think it is fine as is